prog.go

```diff
@@ -631,3 +643,18 @@ func resolveBTFType(name string, progType ProgramType, attachType AttachType) (b
 		return nil, nil
 	}
 }
+
+var haveBpfLSM = internal.FeatureTest("LSM", "5.8", func() (bool, error) {
+	lsm, err := ioutil.ReadFile("/sys/kernel/security/lsm")
+	if err != nil {
+		return false, fmt.Errorf("LSM unavailable: %w", ErrNotSupported)
```

Comment:
The feature test should only return an error if it can't determine whether the feature is supported or not. Is it possible to have LSM enabled but this file not present? Or can we take the absence of the file to mean LSM is not enabled?